atat_provisioning_wizard_api.yaml

```diff
@@ -867,8 +870,17 @@ components:
                 - "in_progress"
                 - "complete"
                 - "failed"
+          required:
```

Comment:
Probably don't need required fields for this or any other object which is never part of a `requestBody`

Reply:
Removed from `PortfolioDraftSummary`.